blog/views.py

```diff
@@ -0,0 +1,68 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.core.paginator import Paginator
+from django.db.models.signals import post_save
+from django.http import HttpRequest, HttpResponse
+from django.shortcuts import render, redirect
+from django.urls import reverse_lazy
+from django.utils.datastructures import MultiValueDict
+from django.views import generic
+from django.views.generic import CreateView
+
+from blog.forms import CommentForm, CustomUserCreationForm
+from blog.models import Post, Commentary
+
+
+def index(request: HttpRequest) -> HttpResponse:
+    post_list = Post.objects.all().order_by("-created_time")
+
+    paginator = Paginator(post_list, 5)
+    page = request.GET.get("page", 1)
+    page_obj = paginator.get_page(page)
+
+    print(
+        f"page: {page}, "
+        f"paginator.num_pages: {paginator.num_pages}, "
+        f"page_obj.object_list: {len(page_obj.object_list)}"
+    )
```

Comment:
Consider removing or commenting out this `print` statement used for debugging purposes to clean up the production code.